PD-1758-update-console-setup-options-list (#3496)
truenas/documentation: content/SCALE/GettingStarted/Install/ConsoleSetupMenuScale.md

  Activating the root user is not recommended.
  {{< /hint >}}

* **5) Create one-time password for "root"**

>   Use to create a one-time password for the "root" user. This is intended for quick authentication to the web interface to further set up secure logins.

Reviewer: in this instance it is log ins, not logins.
```suggestion
  Use to create a one-time password for the "root" user. This is intended for quick authentication to the web interface to further set up secure log ins.
```

* **6) Reset configuration to defaults**

  Use to wipe all system configuration settings and return the system to a fresh install state.

* **7) Open TrueNAS CLI Shell**

  Use to start a shell for running TrueNAS commands, or use the TrueNAS UI **[System Settings > Shell]({{< relref "UseScaleShell.md" >}})**.
  Type `exit` to leave the shell.

* **8) Open Linux Shell**

  Use to start a shell window for running Linux CLI commands.

>   Configuration changes made here are not written to the database and are reset on each system boot.

Reviewer: I think we can fix this legacy content.
```suggestion
  Configuration changes made do not write to the database and reset on each system boot.
```
